testdata/src/main/java/com/hedvig/app/testdata/dashboard/Data.kt

```diff
@@ -1,7 +1,17 @@
 package com.hedvig.app.testdata.dashboard
 
+import com.hedvig.android.owldroid.fragment.ContractStatusFragment
+import com.hedvig.android.owldroid.type.TypeOfContract
 import com.hedvig.app.testdata.dashboard.builders.InsuranceDataBuilder
+import java.time.LocalDate
 
 val INSURANCE_DATA = InsuranceDataBuilder().build()
-
+val INSURANCE_DATA_STUDENT = InsuranceDataBuilder(typeOfContract = TypeOfContract.SE_APARTMENT_STUDENT_BRF).build()
+val INSURANCE_DATA_ACTIVE_AND_TERMINATED = InsuranceDataBuilder(
+    activeStatus = null,
+    activeInFutureAndTerminatedInFutureStatus = ContractStatusFragment.AsActiveInFutureAndTerminatedInFutureStatus(
```

Comment:
We can use the same pattern as we've used in the `HomeDataBuilder` here wrt. contract status - i.e use an enum to switch between the cases.